securedrop/sdconfig.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+import config as _config
+
+MYPY = False
+if MYPY:
+    from typing import List  # noqa: F401
+
+
+class SDConfig(object):
+    def __init__(self):
+        # type: () -> None
+        self.ADJECTIVES = _config.ADJECTIVES
+        self.DATABASE_ENGINE = _config.DATABASE_ENGINE
+        self.DEFAULT_LOCALE = _config.DEFAULT_LOCALE
+        self.FlaskConfig = _config.FlaskConfig
+        self.GPG_KEY_DIR = _config.GPG_KEY_DIR
+        self.JOURNALIST_KEY = _config.JOURNALIST_KEY
+        self.JOURNALIST_TEMPLATES_DIR = _config.JOURNALIST_TEMPLATES_DIR
+        self.JournalistInterfaceFlaskConfig = \
+            _config.JournalistInterfaceFlaskConfig
+        self.NOUNS = _config.NOUNS
+        self.SCRYPT_GPG_PEPPER = _config.SCRYPT_GPG_PEPPER
+        self.SCRYPT_ID_PEPPER = _config.SCRYPT_ID_PEPPER
+        self.SCRYPT_PARAMS = _config.SCRYPT_PARAMS
+        self.SECUREDROP_DATA_ROOT = _config.SECUREDROP_DATA_ROOT
+        self.SECUREDROP_ROOT = _config.SECUREDROP_ROOT
+        self.SESSION_EXPIRATION_MINUTES = _config.SESSION_EXPIRATION_MINUTES
```

Comment:
Note: `SESSION_EXPIRATION_MINUTES` was recently added and may not be defined in many existing `config.py`

Reply:
Okay, I will give another pass on this file and make all of them as default and then if available in `config.py`, then only add here.